src/material-experimental/theming/_custom-tokens.scss

```diff
@@ -510,6 +510,13 @@
 @function slider($systems, $exclude-hardcoded) {
   @return (
     value-indicator-opacity: _hardcode(1, $exclude-hardcoded),
+    value-indicator-width: _hardcode(28px, $exclude-hardcoded),
```

Comment:
Did these all turn out to be necessary? We had discussed maybe getting rid of 1 or 2